src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -0,0 +1,67 @@
+package mate.academy.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_cards")
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @OneToMany
+    @JoinTable(name = "shopping_cards_tickets",
+            joinColumns = @JoinColumn(name = "shopping_card_id"),
+            inverseJoinColumns = @JoinColumn(name = "ticket_id"))
+    private List<Ticket> tickets;
+    @OneToOne
+    @MapsId
+    @JoinColumn(name = "id")
```

Comment:
```suggestion
    private List<Ticket> tickets;
    @OneToOne
    @MapsId
```
Let's remove redundant annotations